docs/spec/spec.md

```diff
@@ -16,7 +16,7 @@ This document details our _Spec_ and _Status_ customizations.
 
 ## kind: Trigger
 
-### group: eventing.knative.dev/v1alpha1
```

Comment:
Can you make a snapshot of v1alpha1, maybe copy this file and make `spec_v1alpha1.md`?

Reply:
ah, yeah - good idea @n3wscott